demos/connector/neo4j/undirected-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -0,0 +1,1117 @@
+{
```

Comment:
<p>Instead of creating a <code>py2neo.Graph</code>  for the second time here, can we create it once at the top and reuse it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Yes, totally agree! I will fix this problem soon!